design/high-level-reactivity.md

```diff
@@ -234,12 +296,20 @@ The new system is designed to avoid this issue, and the implementation will be b
 
 After you analyze the two paths above, there should be a clear path to optimization through careful manual invalidations and disabling reactivity that will let you change a minimum of app code to "twist the knobs" on performance.
 
+// Oh Yeah! I'm seeing huge set of ideas. 
+// Let's implement this. So, we can build a great APM and make money :P
+
 ## Implementation plan
 
 Here's a diagram of all of the pieces we think will need to be built to have a complete system:
 
 ![Block diagram](block-diagram.png)
 
+// What does FlowRouter has to do with this. Do we need to do any changes?
```

Comment:
I just want to emphasize that we want to work with multiple routers. Right now Relay requires you to use Relay Router, we don't want to do that.